src/system/OS_info/os_info_windows.cpp

```diff
@@ -22,6 +22,7 @@
 #define WIN32_LEAN_AND_MEAN
 #include <wbemidl.h>
 #include <windows.h>
+#include <winternl.h>
```

Comment:
I don't think we need this, do we?

Reply:
Unfortunately `NTSTATUS` is defined in there. Or should I define it myself and delete `#include <winternl.h>?

Reply:
Oh, you're right, that's fine, then, keep it. Oddly, it's not listed in https://docs.microsoft.com/en-us/windows/win32/api/winternl/. Ah well.